Makefile

```diff
@@ -64,6 +64,7 @@ requirements: ## install development environment requirements
 	pip install -r requirements/pip.txt
 	pip install -qr requirements/pip-tools.txt
 	pip-sync requirements/dev.txt requirements/private.*
+	pip install . # CLI entry points
```

Comment:
This approach seems to be working OK. My first attempt of `pip install -e .` caused unpredictable failures -- the script stub that was created was different (and broken.) And coverage seems to have stopped being able to see inside the subprocess call, so I had to make some exclusions. But overall I guess the experience is nicer.